pyflakes/checker.py

```diff
@@ -716,6 +716,12 @@ def handleNodeLoad(self, node):
         if not name:
             return
 
+        if name == 'print':
```

Comment:
This needs tests. Further, it looks as though this is running on Python 2 and 3, so someone running pyflakes on Python 2 against Python 2-only code will see a false, positive, right?

Reply:
No the ast structure for `print` is different. In python2 it is a seperate `Node` however in python3, since it is a function it comes under `Name` node.

Reply:
You need to add another bunch of tests without `__future__.print_function` , run only on py27 , to capture their results, and ideally we make those results similar to the errors emitted with print_function enabled.

Reply:
Actually not needed.  Thx